argus/action_manager/windows.py

```diff
@@ -581,10 +603,11 @@ def get_windows_action_manager(client):
     wait_boot_completion(client, username)
 
     # get os type
-    major_version = _get_major_version(client)
+    major_version = introspection.get_os_version(client, 'Major')
```

Comment:
I think, it will be better if you will define two constants: `MAJOR` and `MINOR`, and pass as second argument to this function these defines.